scripts/xtensa-build-zephyr.py

```diff
@@ -192,10 +192,10 @@ def execute_command(command_args, stdin=None, input=None, stdout=None, stderr=No
 	errors=None, text=None, env=None, universal_newlines=None):
 	"""[summary] Provides wrapper for subprocess.run (matches its signature) that prints
 	command executed when 'more verbose' verbosity level is set."""
-	if args.verbose==2:
-		print_args = " ".join(command_args) if isinstance(command_args, list) else command_args
-		print_cwd = f"in: {cwd}" if cwd else f"in: {os.getcwd()}"
-		print(f"Running command: {print_args} {print_cwd}")
+	if args.verbose >= 0:
```

Comment:
I really would hestitate printing all this without verbose mode. If you rebuild something X times and know how this works then it just spams the console for me. I would leave it with at least:
`if args.verbose >= 1:`
But I won't insist if you care, its probably more of a personal preference.